pootle/static/js/editor/components/SuggestionFeedbackForm.js

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (C) Pootle contributors.
+ *
+ * This file is a part of the Pootle project. It is distributed under the GPL3
+ * or later license. See the LICENSE file for a copy of the license and the
+ * AUTHORS file for copyright and authorship information.
+ */
+
+import _ from 'underscore';
+
+import React from 'react';
+
+import FormElement from 'components/FormElement';
+import FormMixin from 'mixins/FormMixin';
+
+export const SuggestionFeedBackForm = React.createClass({
+
+  propTypes: {
+    suggId: React.PropTypes.number.isRequired,
+    initialSuggestionText: React.PropTypes.string.isRequired,
+    localeDir: React.PropTypes.string.isRequired,
+    onAcceptSuggestion: React.PropTypes.func.isRequired,
+    onRejectSuggestion: React.PropTypes.func.isRequired,
+    onChange: React.PropTypes.func.isRequired,
+  },
+
+  mixins: [FormMixin],
```

Comment:
This is unnecessary, as well as anything referring to `this.state.errors`, as any error callbacks are being handled via `PTL.editor.error`.